import User from "../models/user.model.js";
import createError from "../utils/createError.js";

export const deleteUser = async (req, res, next) => {
   try {
    
   } catch (error) {
    next(error)
   }
  
};
export const getUser = async (req, res, next) => {
  try {
    
  } catch (error) {
   next(error)
  }
 
};